components/EventItem.tsx

```diff
@@ -51,50 +48,63 @@ const EventItem = ({ event }: Props) => {
           <Text style={styles.titleText}>{event.title}</Text>
 
           <View style={[styles.location, styles.flexRow]}>
-            <Icon name="location-on" color={event.colorHex} />
+            <MaterialCommunityIcons
+              name="map-marker-outline"
+              size={24}
+              color={event.colorHex}
+            />
             <Text style={styles.infoText}>
               {placemarkToFormattedAddress(event.address)}
             </Text>
           </View>
 
           <View style={styles.flexRow}>
-            <Icon name="event-available" color={event.colorHex} />
+            <MaterialCommunityIcons
+              name="calendar-check-outline"
+              size={24}
+              color={event.colorHex}
+            />
             <Text style={styles.infoText} accessibilityLabel="day">
               {event.duration.formatted()}
             </Text>
           </View>
 
-          <View style={{ paddingVertical: "4%" }}>
-            <Divider style={{ height: 1 }} />
+          <View style={{ marginVertical: 16 }}>
+            <Divider style={{ height: 1, opacity: 0.4 }} />
           </View>
         </View>
 
         {/* People Attending, Distance */}
         <View style={styles.distanceContainer}>
           <View style={[styles.flexRow, { alignItems: "center" }]}>
-            <Icon name="people-alt" color={event.colorHex} />
+            <MaterialCommunityIcons
+              name="account-multiple-outline"
+              size={24}
+              color={event.colorHex}
+            />
             <Text
               style={[styles.attendingText, styles.attendingNumber]}
             >{`${numAttendees}`}</Text>
             <Text style={styles.attendingText}>{" attending"}</Text>
           </View>
 
-          <Shadow distance={4} startColor={shadowColor} offset={[0, 1]}>
-            <View
-              style={[
-                styles.distance,
-                {
-                  backgroundColor: event.colorHex,
-                  borderColor: lightEventColor
-                }
-              ]}
-            >
-              <Icon name="near-me" size={20} color="white" />
-              <Text style={styles.distanceText}>
-                {compactFormatMiles(distance)}
-              </Text>
-            </View>
-          </Shadow>
+          <View
+            style={[
+              styles.distance,
+              {
+                backgroundColor: lightEventColor
+              }
+            ]}
+          >
+            <MaterialCommunityIcons
+              name="navigation-variant-outline"
+              size={22}
```

Comment:
We probably should have the icon size be a multiple of 8 to keep consistency with other components